app/matplotlib.ipynb

```diff
@@ -194,21 +195,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "labels = [\"Product A\", \"Product B\", \"Product C\", \"Product D\"]\n",
     "market_share = [30, 20, 15, 35]\n",
     "\n",
-    "# write your code here"
+    "fig, ax = plt.subplots()\n",
+    "ax.pie(market_share, labels=labels, autopct='%1.1f%%')"
```

Comment:
The pie plot is missing a title and a start angle for better view. You should add a title using `ax.set_title()` and set a start angle using the `startangle` parameter in `ax.pie()`.